configs/platforms/fedora-28-x86_64.rb

```diff
@@ -0,0 +1,11 @@
+platform "fedora-28-x86_64" do |plat|
+  plat.servicedir "/usr/lib/systemd/system"
+  plat.defaultdir "/etc/sysconfig"
+  plat.servicetype "systemd"
+  plat.dist "fc28"
+
+  plat.provision_with "dnf install -y --best --allowerasing autoconf automake rsync gcc make rpmdevtools rpm-libs"
+  ###plat.add_build_repository "http://pl-build-tools.delivery.puppetlabs.net/yum/pl-build-tools-release-#{plat.get_os_name}-28.noarch.rpm"
```

Comment:
Why is this commented out?

Reply:
That was an artifact from when I was doing testing/development with this branch. Just pushed a fix.